.github/workflows/deploy-cloudfront.yml

```
@@ -0,0 +1,41 @@
+name: cloudfront-deployment
+
+on:
+  push:
+    branches: [develop]
+
+jobs:
+  continuous-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)